client/app/bundles/shared/components/Dropdown/Dropdown.scss

```diff
@@ -1,52 +1,49 @@
-@import "../_legacy.scss";
 @import "../_base.scss";
 
-body {
-  font-family: $font-family;
-}
-
-.fill select{
-  background:  #6D0839 url('./down.svg') no-repeat right;
 
-  @include selectDropdown(
-    $padding: 13px 32px,
-    $font-size: $font18
-  );
-  &:focus {
-    outline: none;
-  }
+@mixin dropdown(
+  $padding: 14px 47px 14px 19px,
+  $font-size: $text-size,
+  $color: $input-ghost-text-color,
+  $border-radius: $rounded-corner,
+  $border: none,
+  $appearance: none
+) {
+  padding: $padding;
+  font-size: $font-size;
+  color: $color;
+  border: $border;
+  border-radius: $border-radius;
+  /** TODO: Use PostCSS to add vendor prefixes **/
```

Comment:
Nice!

Reply:
The lazy bum's ethos: do more with less code.